README.md

```diff
@@ -1,26 +1,48 @@
 # patternfly-react
 
-[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
-[![npm version](https://badge.fury.io/js/patternfly-react.svg)](https://badge.fury.io/js/patternfly-react)
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-green.svg?style=for-the-badge)](https://lernajs.io/)
+![PatternFly 3 npm badge](https://img.shields.io/npm/v/patternfly-react.svg?label=PF3%20Core&style=for-the-badge)
+![PatternFly 4 npm badge](https://img.shields.io/npm/v/@patternfly/react-core.svg?label=PF4%20Core&style=for-the-badge)
 
-This project will provide a set of React UI components for the [PatternFly project](https://patternfly.org).
+[PatternFly](https://www.patternfly.org/v4/) is an open source design system built to drive consistency and unify teams. This repo provides a set of React UI components for it.
 
-## Getting Started
+## Intro
 
-[Getting Started with PatternFly 3](./packages/patternfly-3/patternfly-react/README.md)
+PatternFly React is a React Component library offered as PatternFly 3 and PatternFly 4.
 
-[Getting Started with PatternFly 4](./packages/patternfly-4/react-core/README.md)
+### PatternFly 4
+`npm install --save @patternfly/react-core`
 
-## Contributing to PatternFly React
+`yarn add patternfly-react`
 
-### Code of Conduct
+```javascript
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+
+export const MyComponent = props => (
+  <Button>This is an interactive PatternFly 4 styled button!</Button>
+);
+```
+
+### PatternFly 3
+
+`npm install --save patternfly-react`
 
-This project is governed by the Contributor Covenant version 1.4. All contributors and participants agree to abide by its terms. To report violations, send an email to patternfly@redhat.com.
```

Comment:
Why did we remove this?

Reply:
It's still at the bottom, the diff didn't pick it up the best.